python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -382,6 +386,38 @@ class RandomForestClassifier(Base):
 
         return model_protobuf_bytes
 
+    def _tl_model_handles(self, model_bytes):
+        task_category = 1
+        cdef ModelHandle cuml_model_ptr = NULL
+        cdef RandomForestMetaData[float, int] *rf_forest = \
+            <RandomForestMetaData[float, int]*><size_t> self.rf_forest
+        build_treelite_forest(& cuml_model_ptr,
+                              rf_forest,
+                              <int> self.n_cols,
+                              <int> task_category,
+                              <vector[unsigned char] &> model_bytes)
+        mod_handle = <size_t> cuml_model_ptr
+
+        return ctypes.c_void_p(mod_handle).value
+
+    def concatenate_treelite_bytes(self, treelite_handle, deep_check):
+        cdef cumlHandle* handle_ =\
+            <cumlHandle*><size_t>self.handle.getHandle()
+        cdef vector[ModelHandle] *mod_handle_vec \
```

Comment:
Wouldn’t something like model_ handles be a better and more clear variable name here?